lib/python/Plugins/Extensions/Infopanel/QuickMenu.py

```diff
@@ -61,7 +61,7 @@
 else:
 	AUDIOSYNC = False
 
-if path.exists("/usr/lib/enigma2/python/Plugins/SystemPlugins/VideoEnhancement/plugin.pyo"):
+if path.exists("/usr/lib/enigma2/python/Plugins/SystemPlugins/VideoEnhancement/plugin.", getPyExt()):
```

Comment:
Are you sure about "string" , getPyExt().
This looks like a second parameter for path.exists.